src/web/mjs/connectors/NtsVoidScans.mjs

```diff
@@ -1,12 +1,24 @@
 import WordPressMadara from './templates/WordPressMadara.mjs';
-//dead?
 export default class NtsVoidScans extends WordPressMadara {
```

Comment:
Shouldn't this website be removed instead of throwing errors everywhere?

Reply:
There was 2 websites, they rebranded, fused, etc..., its a mess.

I did it like that so people are not lost. Ofc, it can be deleted if you want.

Reply:
Should be rmeoved for the following reasons:
- Existing bookmarks for `ntsvoidscans` will not be preserved/migrated by this approach
- Usually dead websites are removed, so it would be inconsistent to keep just this one in the list
- There is no benefit in keeping a non-functional website, if a user ask in support about `NTS Void Scans` there is the generic answer: Website is dead or user has to provide working url